README.md

```diff
@@ -147,13 +147,13 @@ List of projects that provide terminal user interfaces
 - [pytermgui](https://github.com/bczsalba/pytermgui) A simple yet powerful TUI framework for your Python (3.7+) applications.
 - [pyTermTk](https://github.com/ceccopierangiolieugenio/pyTermTk) self-contained TUI library for Python with a QT-like API semantics
 - [Python Prompt Toolkit](https://github.com/prompt-toolkit/python-prompt-toolkit) Library for building powerful interactive command line applications in Python
-- [ratatui](https://github.com/tui-rs-revival/ratatui) tui-rs revival project
+- [Ratatui](https://github.com/tui-rs-revival/ratatui) A Rust crate for building Terminal UIs (actively maintained fork of tui-rs).
 - [Rich](https://github.com/willmcgugan/rich) is a Python library for rich text and beautiful formatting in the terminal.
 - [tcell](https://github.com/gdamore/tcell) Tcell is an alternate terminal package, similar in some ways to termbox, but better in others.
 - [textual](https://github.com/willmcgugan/textual) is a TUI (Text User Interface) framework for Python inspired by modern web development.
 - [Thermage](https://github.com/thermage/thermage) Thermage is a PHP library that provides a fluent and incredibly powerful, object-oriented interface for customizing CLI output text color, background, formatting, theming and more.
 - [tui-go](https://github.com/marcusolsson/tui-go) A UI library for terminal applications (deprecated)
-- [tui-rs](https://github.com/fdehau/tui-rs) Terminal user interfaces and dashboards using Rust
+- [tui-rs](https://github.com/fdehau/tui-rs) Terminal user interfaces and dashboards using Rust (no longer maintained, use [ratatui](https://github.com/tui-rs-revival/ratatui) instead).
```

Comment:
```suggestion
- [tui-rs](https://github.com/fdehau/tui-rs) Terminal user interfaces and dashboards using Rust (no longer maintained, use Ratatui instead).
```
I don't like having duplicate links on the page because the CI flags it.

Reply:
No problem - committed the suggestion.